tests/unit/test_transformers_next_item_prediction.py

```diff
@@ -0,0 +1,57 @@
+import shutil
+
+import pytest
+from testbook import testbook
+
+from tests.conftest import REPO_ROOT
+
+pytest.importorskip("transformers")
+utils = pytest.importorskip("merlin.systems.triton.utils")
+
+TRITON_SERVER_PATH = shutil.which("tritonserver")
+
+
+@pytest.mark.skipif(not TRITON_SERVER_PATH, reason="triton server not found")
+@testbook(
+    REPO_ROOT / "examples/Next-Item-Prediction-with-Transformers/tf/transformers-next-item-prediction.ipynb",
+    timeout=720,
+    execute=False,
+)
+@pytest.mark.notebook
```

Comment:
This throws a warning (not an error) in the logs:

```
tests/unit/test_transformers_next_item_prediction_with_pretrained_embeddings.py:21
  /home/gha-user/gha1/Merlin/Merlin/tests/unit/test_transformers_next_item_prediction_with_pretrained_embeddings.py:21: PytestUnknownMarkWarning: Unknown pytest.mark.notebook - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html
    @pytest.mark.notebook
```

You can add a `notebook` marker here: https://github.com/NVIDIA-Merlin/Merlin/blob/main/pytest.ini to stop the warning.

Reply:
thank you ver much for looking at this @nv-alaiacano! 🙏 will make the changes today

The GPU CI pipeline that was failing for me the whole day (which I posted about in slack) passed on the last run! (I left it running as I shut down for the day but had no hope it would pass as it failed with the same error for me 5+ times yesterday, but turns out it now passes!)

Thank you very much for looking into this anyhow and for spotting this issue and pointing me how to get rid of the warnings, will do so today 🙂